dev/com.ibm.ws.microprofile.openapi.1.1.model/bnd.bnd

```diff
@@ -0,0 +1,77 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
```

Comment:
Are the copyright dates correct on all these new files?

Reply:
Good point, I will update it to 2019 for 1.1 project